tests/README.md

```diff
@@ -21,3 +21,17 @@ Copy certs from vagrant instance to this repo
 \cp -r /etc/puppetlabs/puppet/ssl/* /vagrant/tests/ssl/
 ```
 
+# Secrets
```

Comment:
Could you add some more text around what files these touch and when you might need to use them.

Reply:
Added additional text. Also updated decrypt script to throw error if needed environment variables are not set.